code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm

```diff
@@ -1,5 +1,6 @@
 //The "BDPtarget" temp visual is created by the expressconsole, which in turn makes two things: a falling droppod animation, and the droppod itself.
-
+#define POD_STANDARD 0
+#define POD_BLUESPACE 1

```

Comment:
You `#define` these three times each, pick one location for them and define them there.